system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml

```
@@ -52,6 +52,9 @@ spec:
     - name: kubeMonitoring.prometheus.prometheusSpec.thanos.objectStorageConfig.existingSecret.key
       value: thanos.yaml
   {{- end }}
+  {{- if index $additionalSettings "extraSpecs" }}
+  {{- index $additionalSettings "extraSpecs" | toYaml | nindent 4}}
+  {{- end }}
```

I'd like to explain the additional extraSpecs here, because at a first glance it reads the same.

```suggestion
  {{/* nested extraSpecs for specific clusters */}}
  {{- if index $additionalSettings "extraSpecs" }}
  {{- index $additionalSettings "extraSpecs" | toYaml | nindent 4}}
  {{- end }}
```